install-guides/README.md

```diff
@@ -0,0 +1,191 @@
+- Untainted master node in cluster.(`kubectl taint nodes $(kubectl get nodes -o=jsonpath='{.items[0].metadata.name}') node-role.kubernetes.io/master:NoSchedule-`) 
+- Go 1.15+ on the system where components are to be built
+
+## Step by step
+The following steps are required to get the Resource Topology Aware Scheduling solution up and running.
+
+1) Configure Kubelet to use Topology Manager, CPU Manager and the Pod Resources endpoint extensions
+2) Install Node Feature Discovery with Topology Exporter
+3) Replace default scheduler with Topology Aware Scheduler enabled Kubernetes scheduler
+4) Create scheduling policy
+
+### Configure Kubelet to use Topology Manager and CPU Manager
+Kubelet requires a CPU Manager policy of static and a Topology manager policy other than 'None' to be set in order to enable Topology Scheduling. It also requires the Pod Resources endpoint, enabled in Kubernetes 1.21, to be enabled through its feature gate.
+
+In kubeadm these variables can be set in a file that sits in 
+- Ubuntu: /etc/default/kubelet
+- CentOS: /etc/sysconfig/kubelet
+
+ Replace the line KUBELET_EXTRA_ARGS with the following:
+
+```
+KUBELET_EXTRA_ARGS=\
+--cpu-manager-policy=static \
+--kube-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
+--system-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
+--topology-manager-policy=single-numa-node \
+--feature-gates="TopologyManager=true,KubeletPodResourcesGetAllocatable=true"
+
+```
+
+The next step is to remove the kubelet cpu checkpoint and restart the service.
+
+```
+systemctl stop kubelet
+systemctl daemon-reload
+rm -f /var/lib/kubelet/cpu_manager_state
+systemctl start kubelet
+```
+To check that the above has take effect run `systemctl status kubelet` and review the arguments passed to kubelet. The above args, including topology manager and cpu manager policies, should now be in place.
+
+### Install Topology Exporter
+In Kubernetes 1.21 a number of enhancements have been made to the Kubelet [to support Topology Scheduling](https://github.com/kubernetes/website/blob/64ef8768ab9ae25a5338a95225a80ac90423473f/content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md#monitoring-device-plugin-resources). These changes are enabled by the feature flag 'KubeletPodResourcesGetAllocatableTrue' and expose information about device topology on a gRPC service running locally on each node. A Resource Topology Exporter can then be used in order to read this information and make it available in the Kubernetes API.
+
+The Resource Topology Exporter is part of Node Feature Discovery. It reads information from the Kubelet's endpoint and creates a representation of the Kubernetes node as a result.
+
+To install Node Feature Discovery from the development branch:
```

Comment:
Might be worth pointing to the relevant NFD PR here.

Reply:
👍